Production_scripts/Process_extracts/Produce_extract_tests/extract_nrs_tests.R

```diff
@@ -0,0 +1,38 @@
+####################################################
+# Name of file - extract_nrs_tests.R
+# Original Authors - Bateman McBride
+# Original Date - July 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               acute processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Deaths", ext = "rds"))
+
+# Read current SLF episode file and filter for 01B and GLS records
+existing_data <- get_existing_data_for_tests(new_data = new_data)
+
+
+# Produce comparison-------------------------------------
+# Compare new file with existing slf data
+comparison <- produce_test_comparison(
+  produce_nrs_extract_tests(existing_data),
+  produce_nrs_extract_tests(new_data)
+)
+
+# Produce Outfile----------------------------------------
+
+# Save test comparisons as an excel workbook
+write_tests_xlsx(comparison, "deaths_extract")
```

Comment:
I think there are already tests for the CHI deaths file (found in: production_scripts/create_slf_lookups), maybe something to check here whether we should specify `nrs_deaths_extract` for the sheet name to make it more specific. Just thinking if we output this to excel and there are more than one deaths sheet we may get confused when they are two different extracts?